runtime/interpreter/value.go

```diff
@@ -186,6 +186,15 @@ func newValueComparator(interpreter *Interpreter, locationRange LocationRange) a
 	}
 }
 
+// ComparableValue
+type ComparableValue interface {
+	Value
```

Comment:
Maybe not necessary, but might prevent some invalid implementations:

```suggestion
	EquatableValue
```